tests/test_custom_attributes.py

```diff
@@ -139,6 +152,22 @@ def test_update_existing_custom_attribute(miq, miq_api_class):
     }
 
 
+def test_compare_custom_attribute_with_section(miq, miq_api_class):
```

Comment:
The tests checks that a new ca with the same name but a different section will be added right?
If so, can you please add a short docstring to explain it?